### PR TITLE
Correct the inherited base compute values

### PR DESCRIPTION
**Pull request by roycollins**

### CODE REVIEW

**Comment by ehlertjd** [comment]
What's the background here?

**Comment by roycollins** [comment]
> What's the background here?

Adjust the region field to be required.   

Also removed the extra config piece as that is done in the inherited class.  
Added a test to confirm that multiple instances of the objects maintain independent configs as that extra config was believed to clobber the other objects.

File: tests/test_providers.py [hunk 1]
```diff
@@ -170,13 +171,63 @@ def test_compute_provider(mocker):
...
-    provider_test.validate()
+    with pytest.raises(ValidationError):
+        provider_test.validate()
     provider_test.config['region'] = ''
     provider_test.validate()
-    provider_test.config['region'] = 'Test Region'
+    provider_test.config['region'] = 'test region'
+
+
+    # Create a second instance to validate isolation of config and attributes
```
Comment (ehlertjd) [inline tests/test_providers.py]: Is that a concern?

Comment (roycollins) [reply tests/test_providers.py]: Ryan thought it might be a problem so I added a test to verify it works.

**Pull request merged**